### PR TITLE
cloud: ovirt change URL in documentation

### PR DESCRIPTION
**Pull request by machacekondra**
##### SUMMARY

Change the URL in documentation types to URL which is publicly available.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

ovirt

##### ANSIBLE VERSION

```
2.3.0
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina

**Comment by machacekondra** [comment]
shipit

File: lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py [hunk 1]
```diff
@@ -129,7 +129,7 @@
     sample: 7de90f31-222c-436c-a1ca-7e655bd5b60c
 affinity_group:
     description: "Dictionary of all the affinity group attributes. Affinity group attributes can be found on your oVirt/RHV instance
-                  at following url: https://ovirt.example.com/ovirt-engine/api/model#types/affinity_group."
+                  at following url: http://ovirt.github.io/ovirt-engine-api-model/master/#types/affinity_group."
```
Comment (resmo) [inline lib/ansible/modules/cloud/ovirt/ovirt_affinity_groups.py]: https works as well just fyi

**Review by resmo — COMMENTED** [review]
https works as well, just fyi

**Comment by resmo** [comment]
shipit

**Pull request merged**